### PR TITLE
More features use indicators in layers panel

### PR DESCRIPTION
**Pull request by DelazJ**

### CODE REVIEW

File: source/docs/user_manual/introduction/general_tools.rst [hunk 1]
```diff
@@ -215,6 +207,19 @@ You may also delete more than one layer or group at once by selecting
...
 This way, all selected layers or groups will be removed from the layers list.
 
+More information on layer or group using indicator icon
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In some circumstances, the :guilabel:`Layers` panel can display next to the
+layer or group an icon to advertize some properties. These symbols are:
+
+* |indicatorFilter| to indicate :ref:`a filter <vector_query_builder>` applied
+  to the layer. Hover over the icon to see the filter expression and double-click
```
Comment (yjacolin) [inline source/docs/user_manual/introduction/general_tools.rst]: `Hover over` it's over :)

File: source/docs/user_manual/introduction/general_tools.rst [hunk 2]
```diff
@@ -215,6 +207,19 @@ You may also delete more than one layer or group at once by selecting
...
+  to update the setting;
+* |indicatorEmbedded| to identify an :ref:`embedded group or layer
+  <nesting_projects>` and the path to their original project file.
+
```
Comment (yjacolin) [inline source/docs/user_manual/introduction/general_tools.rst]: I will add that:
* pensil to indicate that the layer is in edit mode.

File: source/docs/user_manual/introduction/general_tools.rst (hunk 1 above)
Comment (DelazJ) [reply source/docs/user_manual/introduction/general_tools.rst]: Not sure I understand but I think "hover over" (= survoler) is right. Or at least I read it many times in the docs...

File: source/docs/user_manual/introduction/general_tools.rst (hunk 2 above)
Comment (DelazJ) [reply source/docs/user_manual/introduction/general_tools.rst]: Not really the same feature here, but it also indeed indicates particular information on layer so could have a place.

File: source/docs/user_manual/introduction/general_tools.rst (hunk 1 above)
Comment (yjacolin) [reply source/docs/user_manual/introduction/general_tools.rst]: ok, I probably read too fast at first and think there was a typo :)

**Pull request merged**